### ISSUE TITLE
shuffle 不起作用，顺序还是依次读取的

### ISSUE THREAD

**Issue by xiongzhu666** [WZMIAOMIAO/deep-learning-for-image-processing#551]
**System information**
* Have I written custom code:
* OS Platform(e.g., window10 or Linux Ubuntu 16.04):
* Python version:
* Deep learning framework and version(e.g., Tensorflow2.1 or Pytorch1.3):
* Use GPU or not:
* CUDA/cuDNN version(if you use GPU):
* 
* The network you trained: EfficientNetV2

**Describe the current behavior**
shuffle 不起作用，顺序还是依次读取的

**Error info / logs**

**Comment by WZMIAOMIAO** [comment]
我测试pytorch代码，shuffle功能正常。
